### DISCUSSION TITLE
A couple of commits appear to be in the future

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#4211] [General]
GitHub behaves strangely as at the time of writing it is the 6th September, but GitHub shows commits on the 16. September

> 
> Commits on Sep 16, 2024
> 
> Update sandman_fr.ts (340f4ff58e8743602ea8ff12b7e23a5be84005ae)
> DavidXanatos
> DavidXanatos
> committed
> Sep 16, 2024
> ·
> 
> 1.14.7 (09f16661ea2b8d20f37b05dbe4613ddba4d2dc3a)
> DavidXanatos
> DavidXanatos
> committed
> Sep 16, 2024
> ·

Clicking on them shows "DavidXanatos committed next week". Earlier today, this was not the case, everything looked fine. Edit: I might have mistaken that it was displayed just fine, as the last time I looked, the translation update was just committed and I took the displayed "now" for everything being fine.

**Comment by DavidXanatos** [comment]
LOL I was testing some cert stuff and change the clock in my VM, apparently when commiting it takes the system time and github just takes it as is and does not check for plausibility

**Reply by bastik-1001** [reply]
Makes me wonder how far you can go ahead or how early a commit could be.
